designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/definition/package.scala

```diff
@@ -133,7 +133,7 @@ package object definition {
   object UICustomAction {
 
     def apply(action: CustomAction): UICustomAction = UICustomAction(
-      name = action.name,
+      name = action.name.value,
```

Comment:
We could add a custom encoder, decoder. Thanks to that action name would be automatically passed in `UICustomAction` requst. We do the same with `ProcessName`, `ComponentId` and some other value classes. WDYT?